docs/basics.rst

```diff
@@ -48,24 +48,41 @@ There are two ways projects can use ``rapids-cmake`` functions.
 Overriding RAPIDS.cmake
 ***********************
 
-At times projects or developers will need to verify ``rapids-cmake`` branches. To do this you need to override the default git repository and branch that ``RAPIDS.cmake`` downloads, which should be done
-like this:
-
-.. code-block:: cmake
+At times projects or developers will need to verify ``rapids-cmake`` branches. To do this you can set variables that control which repository ``RAPIDS.cmake`` downloads, which should be done like this:
 
+```cmake
```

Comment:
We have to use reStructured Text and not Markdown. See `.. code-block:: cmake` like the previous code had.

Reply:
Updated to use rST syntax